### DISCUSSION TITLE
How to acquire the index of equations picked by model.predict()

### DISCUSSION THREAD

**Discussion by GodfreyL023** [MilesCranmer/PySR#459] [Q&A] [answered]
Hi Miles,
Thanks to your brilliant work, I have had an amazing experience using PySR. 

The method model.sympy() and other export methods need user to input the list of indices of the equations wanted. What makes me confused is that I haven't found any effective way to obtain the indices of those functions picked by model.predict other than searching for them manually in the return of model.fit. Is there any way to get them automatically? 

Appreciate any generous response.

Version
0.16.3

Operating System
Windows

**Comment by MilesCranmer** [comment, marked as answer]
Hi @GodfreyL023,
Thanks, I am glad to hear it is useful for you.

You can do this as follows:

```python
from pysr.sr import idx_model_selection

idx = idx_model_selection(model.equations_, "best")
```
